src/internal_events/nats.rs

```diff
@@ -27,5 +30,10 @@ impl InternalEvent for NatsEventSendError {
         );
         // deprecated
         counter!("send_errors_total", 1);
+        emit!(ComponentEventsDropped {
+            count: 1,
+            intentional: false,
+            reason: reason,
```

Comment:
I think this will work:
```suggestion
            reason,
```

Reply:
Huh, yeah I think so too - I expected rust-analyzer to ding me there though. Double checking before committing the suggestion.